test/Microsoft.TestPlatform.AcceptanceTests/Extension/CompatibilityRowsBuilder.cs

```diff
@@ -56,8 +56,8 @@ public class CompatibilityRowsBuilder
     public bool DebugVSTestConsole { get; set; }
     public bool DebugTestHost { get; set; }
     public bool DebugDataCollector { get; set; }
-    public bool NoDefaultBreakpoints { get; set; } = true;
-
+    public bool DebugStopAtEntrypoint { get; set; }
+    public int? JustRow { get; internal set; }
```

Comment:
Name isn't super easy to gasp. It's to control which row to run, right? Could it be `RunOnlyRow`? Is it worth making this a collection to be able to override the subset of row to run?